module/organizationExtension/helper.js

```
@@ -0,0 +1,166 @@
+		})
+	}
+
+	/**
+	 * Create Organization Extension document
+	 * @method
+	 * @name createOrgExtension
+	 * @param req - requestData
+	 * @returns {Object} Created document
+	 */
+	static async createOrgExtension(req) {
+		return new Promise(async (resolve, reject) => {
```

ESLint

[error] Promise executor functions should not be async.

(no-async-promise-executor)